tests/run-pass/fs.rs

```diff
@@ -21,7 +35,11 @@ fn main() {
     // Reading until EOF should get the whole text.
     file.read_to_end(&mut contents).unwrap();
     assert_eq!(bytes, contents.as_slice());
-    // Removing file should succeed
+
+    // FIXME: Implement stat64 for macos.
```

Comment:
The FIXME should go at the `cfg` above, where things will change when the fix happens.